source/includes/APIReference/CustomFields/_create.md.erb

```diff
@@ -0,0 +1,39 @@
+## Create Custom Fields
+
+<%= partial "includes/APIReference/CustomFields/Examples/create.tmpl.erb" %>
+
+Add one or more customfields.
```

Comment:
Could be good to note here, or in the 'active' property: you can only have 6 active fields at a time.

Reply:
We discussed this in-person. Decided to leave this information in the header block for the customfield object itself.